src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonInTable } from '../PersonInTable';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {people.map(person => (
+        <PersonInTable person={person} />
```

Comment:
key

Reply:
![image](https://user-images.githubusercontent.com/118836537/236577076-7ee93407-bc7a-4790-b1dc-ce114b22fb53.png)
I have it in component